pom.xml

```diff
@@ -678,7 +682,13 @@
         <groupId>org.scalatest</groupId>
         <artifactId>scalatest_${scala.binary.version}</artifactId>
         <version>2.2.1</version>
-        <scope>test</scope>
+        <!--
```

Comment:
Does this have transitive dependencies that might change the overall dependency graph for users of spark-core?